lib/fullcalendar-rails/version.rb

```diff
@@ -1,5 +1,5 @@
 module Fullcalendar
   module Rails
-    VERSION = "2.4.0.2"
+    VERSION = "2.5.0"
```

Comment:
Wrong format on this one.
https://github.com/bokmann/fullcalendar-rails/blob/2.4.0.0/lib/fullcalendar-rails/version.rb for reference.